types/chrome/index.d.ts

```diff
@@ -10205,7 +10205,7 @@ declare namespace chrome {
             address: string;
         }
 
-        export function create(type: string, options?: Object, callback?: (createInfo: CreateInfo) => void): void;
+        export function create(type: string, options?: object, callback?: (createInfo: CreateInfo) => void): void;
```

Comment:
This is a 5-year deprecated socket API. `options` is an interfaceable JSON object (find its declaration [here](https://developer.chrome.com/docs/apps/reference/socket#type-CreateOptions)), but I'd just key-unknown this since it's deprecated.